src/components/Carousel/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { Component } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import aImage from "../../assets/images/a.jpg";
+import bImage from "../../assets/images/b.jpg";
+import cImage from "../../assets/images/c.jpg";
+import "./style.css";
+
+export default class MusicSlider extends Component {
+  render() {
+    const settings = {
+      dots: false,
+      infinite: true,
+      speed: 500,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+      arrows: false,
+    };
+    return (
+      <div className="music-slider">
+        <Slider {...settings}>
+          <div className="slider-image">
+            <img src={aImage} alt="Example" />
+            <div className="slider-content">
+              <h3>Kathmandu</h3>
```

Comment:
this slider content is repeating, make a component and pass its porps. just like to make a button component in koib.